post_installation_configuration/bare-metal-configuration.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+:context: post-install-bare-metal-configuration
+[id="post-install-bare-metal-configuration"]
+= Bare metal configuration
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+When deploying {product-title} on bare metal hosts, there are times when you need to make changes to bare metal hosts either before or after provisioning. This can include inspecting the host's hardware, firmware, and BIOS details. It can also include cleaning disks or changing modifiable BIOS settings. There are two resources that you can use with the Bare Metal Operator (BMO):
```

Comment:
Should be *three* resources and include FirmwareSchema below

Reply:
I think the meaning here is that 2 resources can be changed. It should be added that in order to change HostFirmwareSettings, FirmwareSchema should be used for correct values setting